.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] Unfortunately, the test.md file still exists in your repo even after adding it to the ignore file. Ensure that it doesn't exist anywhere except being listed in the ignore file.